code/modules/jobs/job_types/roguetown/adventurer/types/combat/mage.dm

```diff
@@ -113,10 +113,8 @@
 			H.mind.adjust_skillrank_up_to(/datum/skill/misc/alchemy, 2, TRUE)
 			H.mind.adjust_skillrank_up_to(/datum/skill/magic/arcane, 2, TRUE)
 			H.change_stat("intelligence", 2)
-			H.change_stat("perception", 1)
 			H.change_stat("speed", 3)
-			H.change_stat("constitution", 2)
-			H.change_stat("endurance", 2)
```

Comment:
This is a more personal review but as a spellblade even with these end and con boosts you're not a warrior, magic helps of course but like, not that much unless its pvp

The sword being raised to 3 kinda helps but Ill rather keep the boost and just use a trait to specialize

Reply:
Eh 4 swords might be too much tbh, the mage can still sling out decent spells
Plus 4 is what a specialized warrior gets now